Source/WebKit/WebProcess/InjectedBundle/API/gtk/DOM/WebKitDOMKeyboardEvent.cpp

```diff
@@ -264,9 +264,7 @@ gboolean webkit_dom_keyboard_event_get_alt_graph_key(WebKitDOMKeyboardEvent* sel
 {
     WebCore::JSMainThreadNullState state;
     g_return_val_if_fail(WEBKIT_DOM_IS_KEYBOARD_EVENT(self), FALSE);
-    WebCore::KeyboardEvent* item = WebKit::core(self);
-    gboolean result = item->altGraphKey();
-    return result;
+    return false;
```

Comment:
```suggestion
    return FALSE;
```

(Return type is gboolean, same as int. Different from bool. bool will convert properly to int in this situation, but we try to be strict anyway.)

Reply:
DONE in [b6bfacd](https://github.com/WebKit/WebKit/commit/b6bfacd55a05070624cb47699cb95787edd6bc3b).